CHANGELOG.md

```diff
@@ -18,6 +22,10 @@ IMPROVEMENTS:
 * `vcd_vapp_vm` `disk` has new attribute `size_in_mb` [GH-433]
 * `datasource/*` - all data sources return an error when object is not found [GH-446]
 
+DEPRECATIONS:
+* `resource/vcd_vapp_vm` `network.name` deprecated creation/attaching vApp network when doesn't exist. Requires to create/attach 
```

Comment:
```suggestion
* `resource/vcd_vapp_vm` `network.name` deprecate automatic creation/attaching of a vApp network when it doesn't exist. Requires to create/attach 
```

Also, isn't this deprecation actual only for `type` = `org`? This message needs to be adjusted to reflect that. Now you can read it as if even isolated vApp network is being auto-created, which is not.

Also, the comment looks to be split into two lines needlessly.

Reply:
adjusted